database/mariadb.go

```diff
@@ -0,0 +1,121 @@
+package database
+
+import (
+	"fmt"
+	"strings"
+	"os"
+	"path"
+
+	"github.com/gobackup/gobackup/helper"
+	"github.com/gobackup/gobackup/logger"
+)
+
+// Mariadb database
+//
+// type: mariadb
+// host: 127.0.0.1
+// port: 3306
+// socket:
+// databases:
+// username: root
+// password:
+// args:
+
+type MariaDB struct {
+	Base
+	host          		string
+	port          		string
+	socket        		string
+	databases     		[]string
```

Comment:
By GoBackup current design, we only support dump 1 db for 1 model. 

So you can't change this rule, I can merge this unless you change to 1 database, because it will break the db layer design.

We can move to support multiple databases, but that need to change all databases implementation to support that, not only mariadb.

Reply:
done, backup logic changed to single database.